sessions/forecasting-nowcasting.qmd

```diff
@@ -0,0 +1,798 @@
+  By using the same model structure throughout, the joint approach ensures that assumptions about transmission and reporting are aligned across all time points.
+
+- **What uncertainties does it properly account for?**  
+  The joint model propagates all relevant sources of uncertainty, including those from reporting delays, nowcast estimates, and the evolution of $R_t$.  
+  It also accounts for the correlation between uncertainty in the nowcast and the forecast, which can be important for accurate prediction intervals.  
+  This means that the uncertainty in the most recent data, which is incomplete, is properly reflected in the forecast.
+
+- **Why might it perform better than the pipeline and complete data approaches?**  
+  The joint model can perform better because it uses all available data, including the most recent but incomplete reports, and models the process that generates both the observed and unobserved data.  
+  This allows it to make use of information that the pipeline approach, which separates nowcasting and forecasting, might miss.  
+  By jointly modelling the entire process, it can provide more coherent and potentially more accurate forecasts, especially when the reporting process is well understood.
+
+- **Why might it perform worse than the complete data approach?**  
+  In some situations, the complete data approach, which waits for older data to become nearly complete before forecasting, might perform better.  
+  This can happen if the most recent data are highly incomplete or unrepresentative, for example, due to sudden changes in reporting or the epidemic process.  
+  In such cases, older data may provide a more stable basis for forecasting, even though it is less timely.  
+  It may also underperform if the generative process is not consistent with the data we have. For example, if the growth rate of older data happens to be more consistent with the future growth rate of the epidemic, then the complete data approach will perform better.
+  These issues indicate that the underlying generative process or forecast model could be improved to better capture changes in the epidemic or reporting dynamics.
+
+:::
+
+# Evaluation and comparison
+
+We'll evaluate all four approaches against what will finally be reported (our `complete_at_horizon` dataset) using proper scoring rules.
+
+## Prepare evaluation data
+
+```{r prepare-evaluation}
+# Create evaluation dataset - what we're forecasting against
+evaluation_data <- complete_at_horizon |>
+  filter(day > cutoff, day <= cutoff + forecast_horizon) |>
+  mutate(
+    forecast_horizon = day - cutoff,
+    observed = complete_onsets
+  ) |>
+  select(day, forecast_horizon, observed)
+
+head(evaluation_data)
+```
+
+## Visualise all the forecasts together
+
+As in all other sessions, we first visualise the forecasts together.
+
+```{r plot-all-forecasts}
```

Comment:
I can't really distinguish the overlap/spread of the green/blue lines in particular. Maybe facet? or use sample-derived intervals?